config/application.rb

```diff
@@ -8,6 +8,13 @@
 
 module Todos
   class Application < Rails::Application
+
+    config.middleware.insert_before 0, "Rack::Cors" do
+      allow do
+        origins '*'
+        resource '*', headers: :any, methods: [:get, :post, :options]
```

Comment:
Methods any? Deletes, puts, etc won't work with this right?

Reply:
Just get, post and options; do you want me to change to any?

Reply:
Yeah, no harm right? And todos api supports delete, patch, and put in any case.